tidb-cloud/cli-reference.md

```diff
@@ -65,34 +75,36 @@ If you want to get the properties in one user profile, use [`ticloud config desc
 }
 ```
 
-### Set properties in user profile
+### Set properties in a user profile
 
-Use [`ticloud config set`](tidb-cloud/ticloud-config-set.md) to set properties in the user profile.
+Use [`ticloud config set`](tidb-cloud/ticloud-config-set.md) to set properties in a user profile.
 
 ### Switch to another user profile
 
-Use [`ticloud config use`](tidb-cloud/ticloud-config-use.md) to switch to another user profile. The output is as follows:
+Use [`ticloud config use`](tidb-cloud/ticloud-config-use.md) to switch to another user profile.
+
+An example output is as follows:
 
 ```
 Current profile has been changed to default
 ```
 
-### Open config file with editor
+### Open profile config file with editor
```

Comment:
Using `config file` is better because the config file contains the user profile config and global config(coming later).